### DISCUSSION TITLE
Java 22/23

### DISCUSSION THREAD

**Discussion by 0xC4DE** [itzg/docker-minecraft-server#3153] [Q&A] [answered]
Just curious if Java versions 22/23 will be included any time soon?

**Comment by itzg** [comment, marked as answer]
There's already 12 build variants I'm maintaining, so no, no immediate plans. I prefer to track the recommendation put out by Mojang.

**Reply by 0xC4DE** [reply]
Out of my own curiosity, what all would be involved in using a "custom" java version as the base of any given image? I can imagine all I have to do is modify a few variables, but you would definitely have more insight as to exactly what needs to be done.

**Reply by itzg** [reply]
I have a section for that 😄 

https://docker-minecraft-server.readthedocs.io/en/latest/misc/contributing/building/#building-image-locally-with-alternate-java-base

**Reply by 0xC4DE** [reply]
Appreciated!
